app/models/group.rb

```diff
@@ -15,6 +15,9 @@ class Group < ApplicationRecord
   validates :group_name, presence: true, exclusion: { in: Repository.get_class.reserved_locations }
   validates_uniqueness_of :group_name
   validates_length_of :group_name, maximum: 30
+  validates           :group_name, format: { with: /\A[a-zA-Z0-9\-_ ]+\z/, message: 'group_name must be alphanumeric,
```

Comment:
Don't align this with line 17, just use one space. (You'll also need to fix the alignment of the line below.)